.github/workflows/build.yaml

```diff
@@ -201,6 +201,46 @@ jobs:
           name: ${{matrix.platform}}-lr-logs
           path: '*-logs.tar.gz'
 
+  libtester-deb-install-test:
+    name: libtester deb install test
+    needs: [d, Build, dev-package]
+    if: always() && needs.dev-package.result == 'success'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu20] # Assuming want to do both ubuntu20 and ubuntu22 at some point so setting up with matrix
+    runs-on:  ["self-hosted", "enf-x86-lowtier"] # not sure if this warrants a better machine, but start here
+    container: ${{fromJSON(needs.d.outputs.p)['ubuntu20'].image}}
+    steps:
+      - name: Download cdt
+        uses: AntelopeIO/asset-artifact-download-action@v2
+        with:
+          owner: AntelopeIO
+          repo: cdt
+          file: 'cdt_.*amd64.deb'
+          target: main
+          artifact-name: cdt_ubuntu_package_amd64
+          token: ${{github.token}}
+      - name: Download leap-dev
+        uses: actions/download-artifact@v3
+        with:
+          name: leap-dev-ubuntu20-amd64
+      - name: Install cdt and leap-dev Packages
+        run: |
+          apt install -y ./*.deb
+          apt-get -y install cmake
```

Comment:
you should be able to remove this line. since you're using the cached base image, it already has cmake
https://github.com/AntelopeIO/leap/blob/651d21f2a3a988a019ff58745d125464c310c201/.cicd/platforms/ubuntu20.Dockerfile#L6

Reply:
Good point.